airflow/config_templates/config.yml

```diff
@@ -672,7 +672,7 @@
       version_added: ~
       type: string
       example: ~
-      default: "Airflow HiveOperator task for {{hostname}}.{{dag_id}}.{{task_id}}.{{execution_date}}"
```

Comment:
What's this change for?

Reply:
Since this goes on the default.cfg, which is then `format()`ed fairly sure you don't want this change

Reply:
The double `{` character was used here, so everything should work.
```
>>> "{{ AA }}".format("AA")
'{ AA }'
```

Reply:
Yeahj. I was strugling with this one. Hive operator is wrongly written. It expect the configuration to contain {} references to parameters it passes, but when airflow reads the values from config, replaces it from environment variables (which are missing). So basically setting configuration for hive operator is wrong right now. 

The problem was that Hive Operator when imported tried to read that configuration and since it was missing, it failed. I think I have it right finally (fallback in the operator but no defaults in .yaml/cfg files). Let's see how the tests work it out. Eventually we might want to fix the operator.